tests/strategies/test_strategies_general.py

```diff
@@ -0,0 +1,14 @@
+"""General tests for `oteapi.strategies`."""
+
+
+def test_registered_entry_point_importability():
+    """Ensure all registered entry points (strategies) can be imported."""
+    from oteapi.plugins.factories import StrategyFactory
+
+    package_name = "oteapi"
+
+    for entry_points in StrategyFactory.strategy_create_func.values():
+        for entry_point in entry_points:
+            if entry_point.package != package_name:
```

Comment:
Why is this skipped?

Reply:
We only want to check the entry points representing strategies implemented in _this_ package.
This is not _very_ relevant for the core OTEAPI package, however, this will be very relevant for other plugin packages, and it doesn't harm implementing it already here :)